```
unpacking sources
unpacking source archive /nix/store/yb7qcwalyfkb2mi86yl4ibhad245lsc2-source
source root is source
patching sources
configuring
fixing cmake files...
cmake flags: -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_BUILD_TYPE=Release -DCMAKE_SKIP_BUILD_RPATH=ON -DBUILD_TESTING=OFF -DCMAKE_INSTALL_LOCALEDIR=/nix/store/xra9cdmy8srh1q68x5cdnk54m03dkjch-openttd-jgrpp-0.34.4/share/locale -DCMAKE_INSTALL_LIBEXECDIR=/nix/store/xra9cdmy8srh1q68x5cdnk54m03dkjch-openttd-jgrpp-0.34.4/libexec -DCMAKE_INSTALL_LIBDIR=/nix/store/xra9cdmy8srh1q68x5cdnk54m03dkjch-openttd-jgrpp-0.34.4/lib -DCMAKE_INSTALL_DOCDIR=/nix/store/xra9cdmy8srh1q68x5cdnk54m03dkjch-openttd-jgrpp-0.34.4/share/doc/openttd-jgrpp -DCMAKE_INSTALL_INFODIR=/nix/store/xra9cdmy8srh1q68x5cdnk54m03dkjch-openttd-jgrpp-0.34.4/share/info -DCMAKE_INSTALL_MANDIR=/nix/store/xra9cdmy8srh1q68x5cdnk54m03dkjch-openttd-jgrpp-0.34.4/share/man -DCMAKE_INSTALL_OLDINCLUDEDIR=/nix/store/xra9cdmy8srh1q68x5cdnk54m03dkjch-openttd-jgrpp-0.34.4/include -DCMAKE_INSTALL_INCLUDEDIR=/nix/store/xra9cdmy8srh1q68x5cdnk54m03dkjch-openttd-jgrpp-0.34.4/include -DCMAKE_INSTALL_SBINDIR=/nix/store/xra9cdmy8srh1q68x5cdnk54m03dkjch-openttd-jgrpp-0.34.4/sbin -DCMAKE_INSTALL_BINDIR=/nix/store/xra9cdmy8srh1q68x5cdnk54m03dkjch-openttd-jgrpp-0.34.4/bin -DCMAKE_INSTALL_NAME_DIR=/nix/store/xra9cdmy8srh1q68x5cdnk54m03dkjch-openttd-jgrpp-0.34.4/lib -DCMAKE_POLICY_DEFAULT_CMP0025=NEW -DCMAKE_OSX_SYSROOT= -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_STRIP=/nix/store/fx4kvd2440kgrq73irdkmwqf6vvzwf2m-binutils-2.35.1/bin/strip -DCMAKE_RANLIB=/nix/store/fx4kvd2440kgrq73irdkmwqf6vvzwf2m-binutils-2.35.1/bin/ranlib -DCMAKE_AR=/nix/store/fx4kvd2440kgrq73irdkmwqf6vvzwf2m-binutils-2.35.1/bin/ar -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ -DCMAKE_INSTALL_PREFIX=/nix/store/xra9cdmy8srh1q68x5cdnk54m03dkjch-openttd-jgrpp-0.34.4
CMake Error: The source directory "/build/source" does not appear to contain CMakeLists.txt.
Specify --help for usage, or press the help button on the CMake GUI.
```

--- a/pkgs/games/openttd/jgrpp.nix
+++ b/pkgs/games/openttd/jgrpp.nix
@@ -2,12 +2,12 @@
 
 openttd.overrideAttrs (oldAttrs: rec {
   pname = "openttd-jgrpp";
-  version = "0.34.4";
+  version = "0.40.5";
 
   src = fetchFromGitHub rec {
     owner = "JGRennison";
     repo = "OpenTTD-patches";
     rev = "jgrpp-${version}";
-    sha256 = "125mgia5hgcsn8314xyiip3z8y23rc3kdv7jczbncqlzsc75624v";
+    sha256 = "sha256-g1RmgVjefOrOVLTvFBiPEd19aLoFvB9yX/hMiKgGcGw=";
   };
 })

